package/base-files/files/lib/functions/system.sh

```diff
@@ -96,10 +122,9 @@ mtd_get_mac_text() {
 		return
 	fi
 
-	mac_dirty=$(dd if="$part" bs=1 skip="$offset" count=17 2>/dev/null)
+	mac=$(head -c $((offset + length)) "$part" | tail -c "$length")
```

Comment:
Introduces an additional fork/exec instead of utilizing `dd`'s builtin skipping.